lib/twofishes/result.rb

```diff
@@ -36,21 +24,21 @@ def lng
       feature.geometry.center.lng
     end
 
-    def coordinates
-      [lat, lng]
+    def country_code
+      feature.cc
     end
 
-    def parents
```

Comment:
Because it's heavy, huge and IMHO useless. We could add `response_includes` argument which we would pass to `responseIncludes`, but I don't think it's worth it.
We're breaking backwards compatibility anyway and dropping a few weights more is only good if you ask me. @masone's call.